packages/app/components/image.component.tsx

```diff
@@ -5,9 +5,14 @@ import { Image as ImageBase, ImageStyle, StyleProp } from 'react-native'
 interface ImageProps {
   src: string
   style: StyleProp<ImageStyle>
+  accessibilityIgnoresInvertColors?: boolean
```

Comment:
Should we force this prop to make sure we keep a good level of a11y whenever we use this component?